pyne/material.pyx

```diff
@@ -686,14 +686,14 @@ cdef class _Material:
         return newmat
 
     def collapse_elements(self, nucset):
-        """collapse_elements(self)
-        Collapses the elements in the material, excluding the nucids in 
-	paramater set. This function returns a copy of the material.
+        """collapse_elements(self, nucset)
+        Collapses the elements in the material, excluding the nuclids in 
```

Comment:
I am pretty sure that this was meant to be read as `nuc-ids` indicating that they should be in id-form and not any other format.